.circleci/run-integration-tests.sh

```diff
@@ -7,18 +7,72 @@ BUILD_NUM=$3
 
 echo "Running tests with image '${COLLECTOR_IMAGE}'"
 
-if [[ $remote_host_type != "local" ]]; then
-    # I am not sure why there is a difference in the tests that are run locally and in VMs. Perhaps they should be the same
-    make -C "${SOURCE_ROOT}" integration-tests-repeat-network integration-tests-baseline integration-tests integration-tests-report
+function performance_platform_supported() {
+    if [[ "${VM_TYPE}" == "flatcar" || "${VM_TYPE}" == "cos" ]]; then
+        return 1
+    fi
+    return 0
+}
+
+function integration_tests_with_measurements() {
+    COLLECTOR_BCC_COMMAND="/tools/do_syscall_64.py -o /tmp/baseline.json" make -C "${SOURCE_ROOT}" integration-tests-baseline
+    COLLECTOR_BCC_COMMAND="/tools/do_syscall_64.py -o /tmp/benchmark.json" make -C "${SOURCE_ROOT}" integration-tests-benchmark
+
+    make -C "${SOURCE_ROOT}" integration-tests-repeat-network integration-tests integration-tests-report
+}
+
+function integration_tests_no_measurements() {
+    make -C "${SOURCE_ROOT}" integration-tests-repeat-network \
+                             integration-tests-missing-proc-scrape \
+                             integration-tests-image-label-json \
+                             integration-tests-baseline \
+                             integration-tests-benchmark \
+                             integration-tests \
+                             integration-tests-report
```

Comment:
I'm wondering why we don't have a `ci-integration-tests` target that simply lists all of the required steps as dependencies so we could do:
```sh
make -C "${SOURCE_ROOT}" ci-integration-tests
```